vcpkg/concepts/supported-hosts.md

```diff
@@ -0,0 +1,138 @@
+---
+title:  Supported Hosts
+description: This article describes the platforms on which vcpkg is supported.
+author: bion
+ms.author: bion
+ms.date: 01/16/2024
+ms.topic: concept-article
+---
+
+# Supported Hosts
+
+This document describes the architectures and operating systems supported by vcpkg. The examples
+used in this document are current as of 2024-01-16.
+
+## Dependencies
+
+vcpkg also depends on a few system components to be installed before it can run.
+
+To use vcpkg, a few components need to be installed on the system first.
+
+* Windows: Visual Studio 2015 or later.
+* macOS: XCode Console Tools, tar, git, zip, unzip, and curl. Most ports also need pkg-config.
+We recommend installing these dependencies via Homebrew, such as via a terminal command
+`brew install zip unzip curl pkgconfig`.
+* Linux: A C++ compiler to use, tar, git, zip, unzip, and curl. Most ports also need pkg-config.
+We recommend installing these dependencies with your system package manager.
+  * apt-based platforms: `apt install git curl zip unzip pkgconfig`
+  * rpm-based platforms: `dnf install git curl zip unzip pkgconfig`

```

Comment:
From the issues we know that macOS and Linux users don't get very far without `autoconf autoconf-archive automake make libtool`.

Reply:
And maybe `python3`.